assets/base/i18n/en.json

```diff
@@ -348,34 +347,50 @@
   "mentions.empty.paragraph": "You'll see messages here when someone mentions you or uses terms you're monitoring.",
   "mentions.empty.title": "No Mentions yet",
   "mobile.about.appVersion": "App Version: {version} (Build {number})",
-  "mobile.about.serverVersion": "Server Version: {version} (Build {number})",
-  "mobile.about.serverVersionNoBuild": "Server Version: {version}",
   "mobile.account.settings.save": "Save",
   "mobile.action_menu.select": "Select an option",
   "mobile.add_team.create_team": "Create a new team",
   "mobile.add_team.join_team": "Join Another Team",
```

Comment:
```suggestion
  "mobile.add_team.join_team": "Join another team",
```

Reply:
For anything not related to calls on this PR, plz discuss with UX first then file a ticket